main.tf

```diff
@@ -8,6 +8,27 @@ locals {
   compartment_id = oci_identity_compartment._.id
 }
 
+# Dynamic groups must be created with a tenancy comportament id
```

Comment:
Dynamic groups and identity_policy should be created in the home region.

https://docs.oracle.com/en-us/iaas/Content/Identity/Tasks/managingregions.htm#The